docker/run.sh

```diff
@@ -0,0 +1,12 @@
+# For systems without journald.
+mkdir -p /var/log/journal
+
+# Copy host libsystemd into image to avoid compatibility issues. This operation
+# is only performed if the host's lib is mounted into the container through a 
+# volume such as -v /usr/lib64/:/host/lib/.
+if [ ! -z "$(ls /host/lib/libsystemd* 2>/dev/null)" ]; then
+  rm /lib/x86_64-linux-gnu/libsystemd*
```

Comment:
Would `rm -f` make sense here?

Reply:
Because the container is running as root, I don't feel it's necessary here.